File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/chat_completion.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ChatCompletion:
    """大模型的聊天完成响应结果"""
    id: str
    object: str
    created: int
    model: str
    choices: List[ChatCompletionChoice]
    usage: CompletionUsage = None
    metadata: ChatCompletionMetadata = None
    content: str = None
    agent: PlusAgent = None
    system_fingerprint: str = None

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/ChatCompletion.kt
package com.sdkwork.backend

data class ChatCompletion(
    val id: String? = null,
    val object: String? = null,
    val created: Int? = null,
    val model: String? = null,
    val choices: List<ChatCompletionChoice>? = null,
    val usage: CompletionUsage? = null,
    val metadata: ChatCompletionMetadata? = null,
    val content: String? = null,
    val agent: PlusAgent? = null,
    val systemFingerprint: String? = null
)
